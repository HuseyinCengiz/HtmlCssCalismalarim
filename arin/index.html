<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
  <title>Arin-Modern Free PSD Templateccc</title>

  <!-- Bootstrap -->
  <link href="css/bootstrap.min.css" rel="stylesheet">
  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300i,400" rel="stylesheet">
  <link rel="stylesheet" type="text/css" href="css/font-awesome.min.css">
  <link href="css/main.css" rel="stylesheet">
  <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
  <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script> 
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
      <![endif]-->
    </head>
    <body>
      <header>
        <div class="container">
          <div class="row">
            <a class="logo" href="#"><img src="img/logo.png" alt="arin"/></a>
            <nav class="menu" id="menu">
              <ul>
                <li><a href="#mainContent">HOME</a></li>
                <li><a href="#services">SERVICES</a></li>
                <li><a href="#works">WORKS</a></li>
                <li><a href="#team">TEAM</a></li>
                <li><a href="#contact">CONTACT</a></li>
              </ul>
            </nav>
            <div class="clearfix"></div>
          </div>
        </div>
      </header>
      <!-- MAIN CONTENT-->
      <section id="mainContent" class="text-center">
        <div id="container">
          <div class="mainArea">
            <div id="mainozelarea">
              <h1>arin</h1>
              <h2>MODERN FREE PSD TEMPLATE</h2>
              <P>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, 
                quis nostrud <br/> exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </P>
                <button class="btn btn-success">SCROLL</button>
              </div>
            </div>
          </div>
        </section>
        <!-- SERVICES-->
        <section id="services" class="text-center">
          <div class="container">
            <div class="servicesTop">
              <h1>SERVICES</h1>
              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod <br/> 
                tempor incididunt ut labore et dolore magna aliqua. </p>
              </div>
              <div class="servicesContent">
                <div class="row">
                  <div class="col-md-4">
                    <i class="fa fa-print fa-4x" aria-hidden="true"></i>
                    <h2>Web Design</h2>
                    <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo</p>
                  </div>
                  <div class="col-md-4">
                    <i class="fa fa-desktop fa-4x" aria-hidden="true"></i>
                    <h2>Print Design</h2>
                    <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo</p>
                  </div>
                  <div class="col-md-4">
                    <i class="fa fa-mobile fa-4x" aria-hidden="true"></i>
                    <h2>Mobile Apps</h2>
                    <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo</p>
                  </div>
                </div>
              </div>
            </div>
          </section>
          <!-- WORKS -->
          <section id="works" class="text-center">
           <div class="container">
            <div class="worksTop">
              <h1>WORKS</h1>
              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod <br/> 
                tempor incididunt ut labore et dolore magna aliqua. </p>
              </div>
              <div class="worksContent">
                <div class="row">
                  <div class="col-md-3">
                    <img src="img/work1.jpg" alt="work1"/>
                  </div>
                  <div class="col-md-3">
                    <img src="img/work2.jpg" alt="work1"/>
                  </div>
                  <div class="col-md-3">
                    <img src="img/work3.jpg" alt="work1"/>
                  </div>
                  <div class="col-md-3">
                    <img src="img/work4.jpg" alt="work1"/>
                  </div>
                </div>
              </div>
            </div>
          </section>

          <!-- COUNTER -->

          <section id="counter" class="text-center">
            <div class="container">
              <div class="row">
                <div class="col-md-4">
                  <i class="fa fa-clock-o" aria-hidden="true"></i>
                  <h1>+1286 HOURS OF WORK</h1>
                </div>
                <div class="col-md-4">
                  <i class="fa fa-users" aria-hidden="true"></i>
                  <h1>+94 CLIENTS</h1>
                </div>
                <div class="col-md-4">
                  <i class="fa fa-thumbs-up" aria-hidden="true"></i>
                  <h1>+114 PROJECTS DELIVERED</h1>
                </div>
              </div>
            </div>
          </section>
          <!-- TEAM -->

          <section id="team" class="text-center">
            <div class="container">
              <div class="teamTop">
                <h1>TEAM</h1>
                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod <br/> 
                  tempor incididunt ut labore et dolore magna aliqua. </p>
                </div>
                <div class="teamContent">
                  <div class="row">
                    <div class="col-md-4">
                      <img src="img/team1.png" alt="Kenneth">
                      <h3>Kenneth Anderson</h3>
                      <h4>Managing Director</h4>
                      <i class="fa fa-twitter" aria-hidden="true"></i>
                      <i class="fa fa-linkedin" aria-hidden="true"></i>
                    </div>
                    <div class="col-md-4">
                     <img src="img/team2.png" alt="Sarah">
                     <h3>Sarah Young</h3>
                     <h4>Lead Designer</h4>
                     <i class="fa fa-twitter" aria-hidden="true"></i>
                     <i class="fa fa-linkedin" aria-hidden="true"></i>
                   </div>
                   <div class="col-md-4">
                     <img src="img/team3.png" alt="Dennis">
                     <h3>Dennis Witsel</h3>
                     <h4>Lead Coder</h4>
                     <i class="fa fa-twitter" aria-hidden="true"></i>
                     <i class="fa fa-linkedin" aria-hidden="true"></i>
                   </div>
                 </div>
               </div>
             </div>
           </section>
           <!--PARTNERS-->
           <section id="partners" class="text-center">
             <div class="container">
              <div id="row">
                <div class="col-md-3">
                  <img src="img/partners1.png" alt="EA">
                </div>
                <div class="col-md-3">
                  <img src="img/partners2.png" alt="GAMELOFT">
                </div>
                <div class="col-md-3">
                  <img src="img/partners3.png" alt="CN">
                </div>
                <div class="col-md-3">
                  <img src="img/partners4.png" alt="NSSN">
                </div>
              </div>
            </div>
          </section>
          <!--CONTACT-->
          <section id="contact">
            <div class="container">
              <div class="contactTop text-center">
                <h1>CONTACT</h1>
                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod <br/> 
                  tempor incididunt ut labore et dolore magna aliqua. </p>
                </div>
                <div class="contactContent">
                  <div class="row">
                    <div class="col-md-6">
                     <form>
                       <fieldset>
                         <input type="text" id="name" placeholder="Name">
                       </fieldset>
                       <fieldset>
                         <input type="email" id="email" placeholder="EMail">
                       </fieldset>
                        <fieldset>
                         <textarea rows="5" cols="30" id="message" placeholder="Message"></textarea>
                       </fieldset>
                        <fieldset>
                         <input type="submit" id="submit" value="SEND">
                       </fieldset>
                     </form>
                    </div>
                    <div class="col-md-6">
                    <h2>Our Address</h2>
                    <p>adipiscing elit, sed do eiusmod tempor 
incididunt ut labore et dolore magna aliqua. </p>
<h2>Call Us</h2>
<p>+90 123 45 67 890</p>
<h2>Our Email</h2>
<a mailto:hseyin.cengiz@hotmail.com>Info@hseyin.cengiz</a>
                   </div>
                 </div>
               </div>
             </div>
           </section>



           <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
           <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
           <!-- Include all compiled plugins (below), or include individual files as needed -->
           <script src="js/bootstrap.min.js"></script>
         </body>
         </html>