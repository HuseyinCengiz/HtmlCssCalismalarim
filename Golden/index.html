<!DOCTYPE html>
<html lang="en">
<head>
  <!-- Required meta tags -->
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <!-- Bootstrap CSS -->
  <link rel="stylesheet" type="text/css" href="css/font-awesome.css"/>
  <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css"/> 
  <link rel="stylesheet" type="text/css" href="css/main.css"/>
  <link href="https://fonts.googleapis.com/css?family=Droid+Serif" rel="stylesheet">
  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
  <link href="https://fonts.googleapis.com/css?family=Roboto+Slab" rel="stylesheet">
  <title>Golden Website</title>
</head>
<body>
  <section id="cover">
    <div class="container"> 
      <div class="row">
        <nav class="navbar navbar-expand-lg fixed-top navbar-light">
          <a class="navbar-brand" href="#"><img src="images/logo.png" alt="Golden Website"/> </a>
          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
          </button>
          <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
            <ul class="navbar-nav">
              <li class="nav-item">
                <a class="nav-link" href="#">HOME<span class="sr-only">(current)</span></a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="#">SERVICES</a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="#">PORTFOLİO</a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="#">ABOUT</a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="#">CONTACT</a>
              </li>
            </ul>
          </div>
        </nav>
      </div>
      <div class="row text-center">
       <div class="col-sm-12">
         <h1>Welcome to Our Studio!</h1>
       </div>
       <div class="col-sm-12">
         <h2>IT'S NICE TO MEET YOU</h2>
       </div>
       <div class="col-sm-12">
         <a href="#"><button type="button" class="btn btn-primary">TELL ME MORE</button></a>
       </div>
     </div>
   </div>
 </section>
 <section id="services">
  <div class="container">
    <div class="services-top text-center">
      <h3>SERVICES</h3>
      <p>Proin icalus purun consequat sem cure.</p>
    </div>
    <div class="services-content text-center">
     <div class="row">
       <div class="col-sm-4">
         <div class="item-circle"><i class="fa fa-shopping-cart fa-3x" aria-hidden="true"></i></div>
         <h4>E-Comerce</h4>
         <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit 
          aenean rhoncus posuere odio in tincidunt.</p>
        </div>
        <div class="col-sm-4">
         <div class="item-circle"><i class="fa fa-desktop fa-3x" aria-hidden="true"></i></div>
         <h4>Responsive Web</h4>
         <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit 
          aenean rhoncus posuere odio in tincidunt.</p>
        </div>
        <div class="col-sm-4">
         <div class="item-circle"><i class="fa fa-shield fa-3x" aria-hidden="true"></i></div>
         <h4>Web Security</h4>
         <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit 
          aenean rhoncus posuere odio in tincidunt.</p>
        </div>
      </div>
    </div>
  </div>
</section>
<section id="portfolio">
  <div class="container">
    <div class="portfolio-top text-center">
      <h3>OUR PORTFOLİO</h3>
      <p>Proin icalus purun consequat sem cure.</p>
    </div>
    <div class="portfolio-content text-center">
      <div class="row">
        <div class="col-sm-4">
          <div class="card" style="width: 20rem;">
            <img class="card-img-top" src="images/product1.jpg" alt="Card image cap">
            <div class="card-body">
             <h4 class="card-title">Ebony & Ivory</h4>
             <p class="card-subtitle text-muted">Branding</p>
           </div>
         </div> 
       </div>
       <div class="col-sm-4">
        <div class="card" style="width: 20rem;">
          <img class="card-img-top" src="images/product2.jpg" alt="Card image cap">
          <div class="card-body">
           <h4 class="card-title">Smart Stationary</h4>
           <p class="card-subtitle text-muted">Print Design</p>
         </div>
       </div> 
     </div>
     <div class="col-sm-4">
      <div class="card" style="width: 20rem;">
        <img class="card-img-top" src="images/product3.jpg" alt="Card image cap">
        <div class="card-body">
         <h4 class="card-title">Clever Poster</h4>
         <p class="card-subtitle text-muted">Print Design</p>
       </div>
     </div> 
   </div>
 </div>
 <div class="row">
  <div class="col-sm-4">
    <div class="card" style="width: 20rem;">
      <img class="card-img-top" src="images/product4.jpg" alt="Card image cap">
      <div class="card-body">
       <h4 class="card-title">Vinyl Record</h4>
       <p class="card-subtitle text-muted">Product Mock-Up</p>
     </div>
   </div> 
 </div>
 <div class="col-sm-4">
  <div class="card" style="width: 20rem;">
    <img class="card-img-top" src="images/product5.jpg" alt="Card image cap">
    <div class="card-body">
     <h4 class="card-title">Treehouse Template</h4>
     <p class="card-subtitle text-muted">Web Design</p>
   </div>
 </div> 
</div>
<div class="col-sm-4">
  <div class="card" style="width: 20rem;">
    <img class="card-img-top" src="images/product6.jpg" alt="Card image cap">
    <div class="card-body">
     <h4 class="card-title">Burned Logo</h4>
     <p class="card-subtitle text-muted">Branding</p>
   </div>
 </div> 
</div>
</div>
</div>
</div>
</section>
<section id="aboutus">
  <div class="container">
    <div class="aboutus-top text-center">
      <h3>ABOUT US</h3>
      <p>Proin icalus purun consequat sem cure.</p>
    </div>
    <div class="aboutus-content">
      <div class="row justify-content-start">
        <div class="col-sm-4 text-right">
         <h4>JULY 2010</h4>
         <h4>Our Humble Beginnigs</h4>
         <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
       </div>
       <div class="col-sm-4 text-center">
        <img src="images/aboutus1.png"/>
      </div>
    </div>
    <div class="row justify-content-end">
      <div class="col-sm-4 text-center">
        <img src="images/aboutus2.png"/>
      </div>
      <div class="col-sm-4 text-left">
       <h4>JANUARY 2011</h4>
       <h4>Facing Startup Battles</h4>
       <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
     </div>
   </div>
   <div class="row justify-content-start">
    <div class="col-sm-4 text-right">
      <h4>DECEMBER 2012</h4>
      <h4>Enter The Dark Days</h4>
      <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
    </div>
    <div class="col-sm-4 text-center">
      <img src="images/aboutus3.png"/>
    </div>
  </div>
  <div class="row justify-content-end">
    <div class="col-sm-4 text-center">
      <img src="images/aboutus4.png"/>
    </div>
    <div class="col-sm-4 text-left">
     <h4>FEBRUARY 2014</h4>
     <h4>Our Triumph</h4>
     <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
   </div>
 </div>
 <div class="row justify-content-center">
  <div class="col-sm-4">
    <div class="about-circle">
      <p class="yellow-text">OUR <br>
        STORY <br>
        CONTINUES<br>
        ...
      </p>
    </div>
  </div>
</div>
</div>
</div>
</section>
<section id="team">
  <div class="container">
    <div class="team-top text-center">
      <h3>OUR AMAZİNG TEAM</h3>
      <p>Proin icalus purun consequat sem cure.</p>
    </div>
    <div class="team-content">
      <div class="row">
        <div class="col-sm-4 text-center">
          <div class="team-circle"></div>
          <h4>Kimberly Thompson</h4>
          <h5>Marketer</h5>  
          <ul class="social-icons">
            <li class="social-icon"><i class="fa fa-twitter" aria-hidden="true"></i></li>
            <li class="social-icon"><i class="fa fa-facebook" aria-hidden="true"></i></li>
            <li class="social-icon"><i class="fa fa-pinterest" aria-hidden="true"></i></li>
          </ul>
        </div>
        <div class="col-sm-4 text-center">
          <div class="team-circle"></div>
          <h4>Rico Massimo</h4>
          <h5>Coder</h5>  
          <ul class="social-icons">
            <li class="social-icon"><i class="fa fa-twitter" aria-hidden="true"></i></li>
            <li class="social-icon"><i class="fa fa-facebook" aria-hidden="true"></i></li>
            <li class="social-icon"><i class="fa fa-pinterest" aria-hidden="true"></i></li>
          </ul>
        </div>
        <div class="col-sm-4 text-center">
          <div class="team-circle"></div>
          <h4>Uku Mason</h4>
          <h5>Graphic Designer</h5>  
          <ul class="social-icons">
            <li class="social-icon"><i class="fa fa-facebook" aria-hidden="true"></i></li>
            <li class="social-icon"><i class="fa fa-pinterest" aria-hidden="true"></i></li>
          </ul>
        </div>
      </div>
      <div class="row">
        <p class="text-center">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.Proin iaculis purus consequat sem cure digni ssim.</p>
      </div>
    </div>
  </div>
</section>
<section id="partner">
  <div class="container">
    <div class="row text-center">
      <div class="col-sm-3">
        <img src="images/partner1.jpg" alt="Envato">
      </div>
      <div class="col-sm-3">
        <img src="images/partner2.jpg" alt="WordPress">
      </div>
      <div class="col-sm-3">
        <img src="images/partner3.jpg" alt="Tuts+">
      </div>
      <div class="col-sm-3">
        <img src="images/partner4.jpg" alt="MicroLancer">
      </div>
    </div>
  </div>
</section>
<section id="contact">
  <div class="container">
   <div class="contact-top text-center">
    <h3>CONTACT US</h3>
    <p>Proin icalus purun consequat sem cure.</p>
  </div>
  <div class="contact-content">
    <form class="golden-form" role="form" method="post">
      <div class="row">
        <div class="col-sm-6">
          <div class="form-element">
            <input type="text" class="golden-form-element" id="name" name="name" placeholder="YOUR NAME"/>
          </div> 
          <div class="form-element">
            <input type="email" class="golden-form-element" id="email" name="email" placeholder="YOUR E-MAIL"/>
          </div> 
          <div class="form-element">
            <input type="text" class="golden-form-element" id="subject" name="subject" placeholder="SUBJECT"/>
          </div> 
        </div>
        <div class="col-sm-6">
         <div class="form-element">
           <textarea class="golden-form-element" name="message" placeholder="YOUR MESSAGE"></textarea>   
         </div> 
       </div>
     </div>
     <div class="row justify-content-center">
       <div class="form-element">
         <input type="submit" class="btn btn-primary" id="submit" name="submit" value="SEND MESSAGE"/>
       </div>
     </div>
   </form>
 </div>
</div>
</section>
<section id="footer">
  <div class="container">
    <div class="row">
      <div class="col-sm-4">
        <p>&copy; Copyright 2017 FreebiesXpress.com</p>
      </div>
      <div class="col-sm-4">
        <ul class="company-social-icons text-center">
      <li class="company-social-icon"><i class="fa fa-twitter" aria-hidden="true"></i></li>
      <li class="company-social-icon"><i class="fa fa-facebook" aria-hidden="true"></i></li>
      <li class="company-social-icon"><i class="fa fa-pinterest-p" aria-hidden="true"></i></li>
      <li class="company-social-icon"><i class="fa fa-google-plus" aria-hidden="true"></i></li>
         </ul> 
     </div>
   </div>
 </div>
</section>
<!-- jQuery first, then Popper.js, then Bootstrap JS -->
<script src="js/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.2.0/js/tether.min.js" integrity="sha384-Plbmg8JY28KFelvJVai01l8WyZzrYWG825m+cZ0eDDS1f7d/js6ikvy1+X+guPIB" crossorigin="anonymous"></script>
<script src="js/bootstrap.min.js"></script>
<!-- Optional JavaScript -->
<script src="js/main.js"></script>
</body>
</html>